### ISSUE TITLE
PDP1 Assembler

### ISSUE THREAD

**Issue by Phi-Ho** [MiSTer-devel/PDP1_MiSTer#3]
Greetings,

I found PDP1_MiSTer\pong\pong.asm and PDP1_MiSTer\paper_tapes\pong.rim.

Is there an assembler to go from pong.asm to pong.rim

Thank you for your help.

Regards,

phiho

**Comment by hrvach** [comment]
Yes, it runs as a program on the pdp-1 itself. However, it's much easier to use the ported one and run it on Linux.

Check this out: https://github.com/simh/simtools

There is even a Debian package in experimental :) 

Good luck!

**Comment by Phi-Ho** [comment]
Hi,

Would you please give me a link and instructions to use it on PDP1_MISTer.

Thank you.

Regards,

phiho

**Comment by hrvach** [comment]
The repo is at https://github.com/simh/simtools, you clone it using:

git clone https://github.com/simh/simtools.git
then cd simtools, and make

This will create a simtools/crossassemblers/macro1/ folder that contains the compiled macro1 binary you use to translate an assembly language source into .rim file that can be loaded into the pdp-1.

Please note none of this is terribly user-friendly but I'm sure you'll get the hang of it after investing some time. 

Good luck!

**Comment by Phi-Ho** [comment]
Hi,

Thanks a lot for the link. I will look into it.

Regards,

phiho
